libs/client-api-for-wasm/src/logger.rs

```diff
@@ -0,0 +1,31 @@
+use crate::wasm_trace;
+
+pub struct WASMLogger;
+
+impl log::Log for WASMLogger {
```

Comment:
using `tracing-wasm` that can reuse all existing logs in the crate.( can take a look at appflowy_web/wasm-libs/af-wasm)